src/components/Timeline/src/bem-mapping.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * This file overrides the default Material UI styling by mapping their style
+ * _'keys'_ to ours using the `classes` attribute.
+ *
+ * Some components also use a `className` attribute. This is defined in the
+ * component's typescript file.
+ */
+
+export const timelineClasses = {
+  root: 'denhaag-timeline',
+};
+
+export const stepClasses = {
+  root: 'denhaag-timeline__step',
+  completed: 'denhaag-timeline__step--completed',
+};
+
+export const stepIconClasses = {
+  root: 'denhaag-timeline__step-icon',
+  colorAction: 'denhaag-timeline__step-icon--active',
+  colorSecondary: 'denhaag-timeline__step-icon--completed',
+  colorPrimary: 'denhaag-timeline__step-icon--inactive',
+  colorDisabled: 'denhaag-timeline__step-icon--disabled',
+};
+
+export const stepCollapseIconClasses = {
+  root: 'denhaag-timeline__step__collapse-icon',
```

Comment:
`denhaag-timeline__step-collapse-icon`, or `denhaag-timeline__step_collapse-icon` (up to you which you prefer, try to be consistent in whichever solution you go for here and might be good to discuss this with the team)

Reply:
done

Reply:
Same here, doesn't actually seem to be resolved?

Reply:
Yes, I forgot to push the changes on my laptop.